ghinsights/USQL/StageData.usql

```diff
@@ -17,7 +17,7 @@ DECLARE @IngestDatePartition = @IngestDate.ToString("yyyy-MM-dd");
     EXTRACT  EventName string
             ,IngestDate DateTime
             ,Data SqlMap<string, byte[]>
-    FROM "wasb://raw@ghinsightspublic.blob.core.windows.net/{EventName:*}/v1/{IngestDate:yyyy}/{IngestDate:MM}/{EventName:*}_{IngestDate:yyyy}_{IngestDate:MM}_{IngestDate:dd}.json.gz"
+    FROM "wasb://raw@ghinsightsms.blob.core.windows.net/{EventName:*}/v1/{IngestDate:yyyy}/{IngestDate:MM}/{EventName:*}_{IngestDate:yyyy}_{IngestDate:MM}_{IngestDate:dd}.json.gz"
```

Comment:
All the other changes look good, but don't merge in the change to StageData.usql.